ext/node/polyfills/process.ts

```diff
@@ -441,11 +441,7 @@ Process.prototype.config = {
 };
 
 Process.prototype.cpuUsage = function () {
-  warnNotImplemented("process.cpuUsage()");
-  return {
-    user: 0,
-    system: 0,
-  };
+  return Deno.cpuUsage();
```

Comment:
Small nit, but maybe define this the same way that `memoryUsage` is defined. That is, create a `function` and then just `Process.prototype.memoryUsage = memoryUsage;`.

I was going to suggest moving this next to `memoryUsage` too, but it seems that the `prototypes` are mostly organized in alphabetical order (not all), so maybe leave it here.